src/types/CustomError.ts

```diff
@@ -0,0 +1,4 @@
+export type CustomError = {
+  active: boolean,
+  text: string,
+};
```

Comment:
would be better to use `enum `and list all possible errors with appropriate messages